tools/build_plugins.sh

```diff
@@ -10,7 +10,20 @@ build_plugin() {
         cd $TMPDIR
         touch $plugin.ml
         bapbuild -package bap-plugin-$plugin $plugin.plugin
-        bapbundle update -desc "`ocamlfind query -format "%D" bap-plugin-$plugin`" $plugin.plugin
+        DESC=`ocamlfind query -format "%D" bap-plugin-$plugin`
+        TAGS=`ocamlfind query -format "%(tags)" bap-plugin-$plugin`
+        if [ -z "$TAGS" ]; then
+            bapbundle update -desc "$DESC" $plugin.plugin
+        else
+            bapbundle update -desc "$DESC" -tags "$TAGS" $plugin.plugin
+        fi
+        if [ -f $plugin/resources ]; then
```

Comment:
This is already removed from the master branch, check your merge. It is strange, why it adds it back.